roles/horizon/templates/etc/openstack-dashboard/local_settings.py

```diff
@@ -59,7 +59,7 @@
         'fade_duration': 1500,
         'types': ['alert-success', 'alert-info']
     },
-    'help_url': "http://docs.openstack.org/kilo/",
```

Comment:
You reverted change that was made to fix the help url to pin to kilo. Not sure if this was intentional.